README.md

```diff
@@ -31,21 +37,25 @@ Hudi manages the storage of large analytical datasets on DFS (Cloud stores, HDFS
 Hudi provides the ability to query via three types of views:
  * **Read Optimized View** - Provides excellent snapshot query performance via purely columnar storage (e.g. [Parquet](https://parquet.apache.org/))
  * **Incremental View** - Provides a change stream with records inserted or updated after a point in time.
- * **Real-time View** - Provides snapshot queries on real-time data, using a combination of columnar & row-based storage (e.g Parquet + [Avro](http://avro.apache.org/docs/current/mr.html))
+ * **Real-time View** - Provides snapshot queries on real-time data, using a combination of columnar & row-based storage (e.g [Parquet](https://parquet.apache.org/) + [Avro](http://avro.apache.org/docs/current/mr.html))
 
 Learn more about Hudi at [https://hudi.apache.org](https://hudi.apache.org)
 
-### Building Apache Hudi from source {#building-hudi}
+## Building Apache Hudi from source
```

Comment:
this is linked from the quickstart page.. can we keep the `{#building-hudi}`?

Reply:
It looks  strange here if we keep. I'll update the link info at the quickstart page, change 
`https://github.com/apache/incubator-hudi#building-apache-hudi-from-source-building-hudi`
to `https://github.com/apache/incubator-hudi#building-apache-hudi-from-source`. WDYT?

Reply:
I prefer to leave this alone. Can you please revert? IMO `building-hudi` is simpler/shorter url

Reply:
> I prefer to leave this alone. Can you please revert? IMO `building-hudi` is simpler/shorter url

no problem, done.